build_tools/pytype/check_diff.sh

```diff
@@ -10,15 +10,21 @@
 #   Defaults to comparing against 'main'.
 #     ./build_tools/pytype/check_diff.sh
 #   A specific branch can be specified.
-#     ./build_tools/pytype/check_diff.sh  google
+#     ./build_tools/pytype/check_diff.sh  some-other-branch
 #   Or all python files outside of './third_party/' can be checked.
 #     ./build_tools/pytype/check_diff.sh  all
 
 DIFF_TARGET="${1:-main}"
 echo "Running pycheck against '${DIFF_TARGET?}'"
 
 if [[ "${DIFF_TARGET?}" = "all" ]]; then
-  FILES=$(find -name "*\.py" -not -path "./third_party/*")
+  # Exclude third_party files and some directories that we add to .gitignore
+  FILES=$(find -name "*\.py" \
+          -not -path "./third_party/*" \
+          -not -path ".venv/*" \
+          -not -path "*.venv/*" \
+          -not -path "./build/*" \
+          -not -path "./build-*/*")
```

Comment:
As it turns out, trying to lint venv installed scripts and the build directory can be slow...